action.yml

```
@@ -6,5 +6,5 @@ branding:
   color: 'blue'
 
 runs:
-  using: 'node12'
+  using: 'node16'
```

Per my comment (https://github.com/Submitty/Submitty/issues/9895#issuecomment-1768531950), node 16 is EOL and Node 20 hits LTS next week, that I'd just go with Node 20 to be forward facing as possible, and not require a new PR in some limited amount of time.

```suggestion
  using: 'node20'
```